developer-concepts/readme.adoc

```diff
@@ -837,9 +841,9 @@ Deleting a job deletes all the pods as well. Delete the job as:
 
 === Pre-requisites
 
-For Kubernetes cluster versions < 1.8, Cron Job is created with API version `batch/v2alpha1`. This version needs to be explicitly enabled in the Kubernetes cluster as explained in link:cluster-install#turn-on-an-api-version-for-your cluster[Turn on an API version for your cluster].
+For Kubernetes cluster versions < 1.8, Cron Job is created with API version `batch/v2alpha1`. This version needs to be explicitly enabled in the Kubernetes cluster as explained in link:../cluster-install#turn-on-an-api-version-for-your-cluster[Turn on an API version for your cluster].
 
-`batch/v2alpha1` API version for Cron Job is deprecated for the cluster version >= 1.8. You should switch to `batch/v1beta1`. This version is enabled by default in the API server.
```

Comment:
I think its still good to clarify that there is no need to turn this version on, wdyt?

Reply:
Let me come back on this because I think as I started testing this tutorial, my default cluster was at 1.7.2 and I thought I had to switch API 'batch/v1beta1' as I ran into the error but I never got it to working because of my rolling-update situation. I will re-run the tests later afternoon and just go over cronjob tutorial and provide an update

Reply:
Sounds good, will wait for your input before merging